packages/monitoring-system/data-processor/package.json

```diff
@@ -20,15 +21,17 @@
   "author": "asonawalla@google.com",
   "license": "Apache-2.0",
   "dependencies": {
-    "express": "^4.17.1",
-    "firebase-admin": "^9.0.0"
+    "@google-cloud/firestore": "^4.1.1",
```

Comment:
Using the `@google-cloud` version of the Firestore client for now as @bcoe recommended, but @chingor13 suggested that we might want to use the `firebase-admin` version to be consistent with the client we'll use for the web app/browser. 

Either way, it should be easy to swap one for the other at a later stage